rest_api/sawtooth_rest_api/route_handlers.py

```diff
@@ -834,6 +834,18 @@ def _set_wait(self, request, validator_query):
                 validator_query.timeout = int(self._timeout * 0.95)
 
     @staticmethod
+    def _format_statuses(statuses):
+        """Reformat converted BatchStatus dicts: drop empty keys, rename 'id'.
```

Comment:
Perhaps a note that this mutates the status list - returning the value makes it appear like it's creating a new list. Or, just drop the return value, since it looks like it should be properly changed without it.